rental.rb

```diff
@@ -0,0 +1,10 @@
+require './book'
+class Rental
+  def initialize(date)
+    @date = date
+    @person = person
+    book.rentals << self
+    person.rentals << self
+  end
```

Comment:
- [ ] The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. Link to the example article.
- [ ]  Make sure The rental is adding itself to the @book achieving the 'belongs-to' association

```ruby
  def initialize(date, book, person)
    ...
     @book = book
    book.rentals << self
    ...
```

Reply:
That is correct I will add the right arguments to the method @BenMukebo

Reply:
Well resolved @d4nQw3rty 👍🏽